BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php

```diff
@@ -90,4 +98,28 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
         return self::ACCESS_ABSTAIN;
     }
+
+    /**
+     * @param NodeInterface  $node
+     * @param GroupInterface $group
+     * @param string         $attribute
+     *
+     * @return boolean
+     */
+    protected function isGrantedNodeGroupRole(NodeInterface $node, GroupInterface $group, $attribute)
+    {
+        $nodeGroupRole = $group->getNodeRoleByNodeAndRole($node->getNodeId(), $attribute);
+        if ($nodeGroupRole instanceof NodeGroupRoleInterface) {
+            if ($nodeGroupRole->isGranted() === 'inherit') {
```

Comment:
Is it normal that the method isGranted returns something else than true or false ?
In my mind the voter should do the recursive work to only answer true or false. By saying inherit, it doesn't answer the question to my mind.